ts/store/actions/onboarding.ts

```diff
@@ -8,24 +8,20 @@ import {
   createStandardAction
 } from "typesafe-actions";
 
-export const tosAccept = createAsyncAction(
-  "TOS_ACCEPT_REQUEST",
-  "TOS_ACCEPT_SUCCESS",
-  "TOS_ACCEPT_FAILURE"
-)<void, void, void>();
-
 export const fingerprintAcknowledge = createAsyncAction(
   "FINGERPRINT_ACKNOWLEDGE_REQUEST",
   "FINGERPRINT_ACKNOWLEDGE_SUCCESS",
   "FINGERPRINT_ACKNOWLEDGE_FAILURE"
 )<void, void, void>();
 
+export const tosAccepted = createStandardAction("TOS_ACCEPTED")();
```

Comment:
```suggestion
export const tosAccepted = createStandardAction("TOS_ACCEPTED")<number>();
```
For debug purposes I think it would be useful to know which ToS version user accepted.
So also **_analytics.ts_** should be updated
```typescript
case getType(tosAccepted):
      return mp.track(action.type, {
        tosVersion: action.payload
      });
```